backend/src/main/kotlin/hu/bme/sch/cmsch/util/Utility.kt

```diff
@@ -29,15 +32,17 @@ class DI(
     }
 }
 
+private fun getFileStoragePath(): String = if (!DI.instance.startupPropertyConfig.external.startsWith("/")) {
+    System.getProperty("user.dir") + "/" + DI.instance.startupPropertyConfig.external
```

Comment:
I'd just use the working directory for root and not the `user.home`

Reply:
I moved this code from the file upload function, I don't exactly understand why it works like this, so I didn't want to change it